verification/verify_cycle_extrapolation.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python3
+
+import sys
+sys.path.append('..')
+
+import numpy as np
+from neml import interpolate,solvers, drivers,models, elasticity, ri_flow, hardening, surfaces, visco_flow, general_flow, creep, damage
+
+import matplotlib.pyplot as plt
+import time
+
+if __name__ == "__main__":
+
+    E = 150000
+    nu= 0.3
+
+    n = 9.282
+    eta = 228.09
+    sY = 7.485
+
+    Q = 153.9
+    b = 1.9878
+
+    C1 = 15118
+    g1 = 309.63
+
+    C2 = 2782.8
+    g2 = 18.239
+
+    surface = surfaces.IsoKinJ2()
+    iso = hardening.VoceIsotropicHardeningRule(sY, Q, b)
+    cs = [C1, C2]
+    gs = np.array([g1, g2])
+    gmodels = [hardening.ConstantGamma(g) for g in gs]
+    hmodel = hardening.Chaboche(iso, cs, gmodels, [0.0] * len(cs),[1.0] * len(cs))
+
+    fluidity = visco_flow.ConstantFluidity(eta)
+
+    vmodel = visco_flow.ChabocheFlowRule(surface, hmodel, fluidity, n)
+
+    mu = E/(2*(1+nu))
+    K = E/(3*(1-2*nu))
+
+    elastic = elasticity.IsotropicLinearElasticModel(mu, "shear", K, "bulk")
+    fmodel = general_flow.TVPFlowRule(elastic, vmodel)
+    bmodel = models.GeneralIntegrator(elastic, fmodel, verbose = False)
+    A          = 1e4
+    xi         = 4.0
+    phi        = 2.0
+
+    model = damage.ModularCreepDamageModel_sd(elastic, A, xi, phi,damage.VonMisesEffectiveStress(),bmodel)
+
+    emax    = np.array([ 2.0  , 2.0  , 1.0   , 2.0  ])/200.0
+    R       = np.array([ -1.0 , -1.0 , -1.0  , -1.0 ])
+    erate   = np.array([ 0.004, 0.004, 0.004 , 0.004])
+    hold_t  = np.array([ 0.0  , 6.0  , 30.0  , 6.0  ])*60.0
+    hold_c  = np.array([ 0.0  , 0.0  ,  0.0  , 6.0  ])*60.0
+    ncycles = np.array([ 500  , 500  , 200   , 120  ])
```

Comment:
add some comment in what you are trying to verify and what kind of cycles you are setting up
I assume from this you have 4 types of cycles that are going to concatenated one after another 
Also, add at least one case where you have hold during compression

Reply:
There is no concatenation. We are just comparing 4 different cyclic tests results (max,min, vs. cycles; & computation time) with and without extrapolation. The last 4th case has hold in compression. I don't think any comment is necessary here, it should be straightforward, especially with the plot titles.